src/bridge/ERC20Outbox.sol

```diff
@@ -22,6 +52,18 @@ contract ERC20Outbox is AbsOutbox {
         return AMOUNT_DEFAULT_CONTEXT;
     }
 
+    /// @inheritdoc AbsOutbox
+    function _getAmountToUnlock(uint256 value) internal view override returns (uint256) {
+        // make sure that inflated amount does not overflow uint256
+        if (nativeTokenDecimals > 18) {
+            if (value > MAX_BRIDGEABLE_AMOUNT) {
```

Comment:
should swap these since MAX_BRIDGEABLE_AMOUNT is a constant (cheaper to access)

also can cache nativeTokenDecimals since it as accessed again in L64